app/views/microposts/new.js.erb

```diff
@@ -1,7 +1,4 @@
-<% if logged_in? %>
-alert("Hii please proceed..");
-<%else%>
-alert("Sorry something went wrong");
-<%end%>
-var blogs = document.querySelector("#blogs");
-blogs.insertAdjacentHTML("beforeend", "<%= j render 'form', micropost: @micropost %>");
+$(document).ready(function(){
```

Comment:
@HrisikeshSinghaChowdhury why do we need $(document).ready here?

Reply:
Ok i got it